src/hooks/use-event-actions.js

```diff
@@ -246,12 +259,14 @@ export const editAppointment = (event, context, t) => ({
 	disabled: !event.permission,
 	click: (ev) => {
 		if (ev) ev.stopPropagation();
-		context.utils.setEvent(event);
-		context.utils.setAction(EventActionsEnum.EDIT);
+		context.replaceHistory(
+			`/${event.resource.calendar.id}/${EventActionsEnum.EDIT}/${event.resource.id}/${event.resource.ridZ}`
+		);
 	}
 });
 
-export const moveInstanceToTrash = (event, context, t) => ({
+/* TODO: delete after the implementation of the dropdown to distinguish between series and instances
```

Comment:
Did not got your Point .I guess we already have a dropdown for showing different actions related to series and instance